### PR TITLE
Fix 277 update readme and landing page

### LINKED ISSUE #277: Update CUQIpy readme example to show the cookie example from the paper

### PR DESCRIPTION
**Pull request by nabriis**
Closes #277

### CODE REVIEW

File: cuqi/data/_data.py [hunk 1]
```diff
@@ -350,11 +350,13 @@ def cookie(size=128, grayscale=True):
     # Convert to grayscale
     if grayscale:
         cookie = rgb2gray(cookie)
-        cookie[cookie < 0.05] = 0 # Make background completely black
-
+        
     # Resize
     cookie = imresize(cookie, size)
 
+    if grayscale:
```
Comment (jakobsj) [inline cuqi/data/_data.py:357]: What's going on here - I thought this was done in another PR?

**Review by jakobsj — CHANGES_REQUESTED** [review]
Looks very good! A few small comments. The ones to the readme probably also apply to the index.rst.

File: README.md [hunk 2]
```diff
@@ -6,70 +6,82 @@
...
+<img src="https://cuqi-dtu.github.io/CUQIpy/_images/deconv2D_exact_sol.png" alt="Sharp image (exact solution)" width="330">
+<img src="https://cuqi-dtu.github.io/CUQIpy/_images/deconv2D_data.png" alt="Blurred and noisy image (data)" width="330">
+<img src="https://cuqi-dtu.github.io/CUQIpy/_images/deconv2D_post_mean.png" alt="Estimated image (posterior mean)" width="330">
+<img src="https://cuqi-dtu.github.io/CUQIpy/_images/deconv2D_post_std.png" alt="Uncertainty (posterior standard deviation)" width="330">
+<img src="https://cuqi-dtu.github.io/CUQIpy/_images/deconv2D_noise_level.png" alt="Noise level (posterior trace)" width="660">
+<img src="https://cuqi-dtu.github.io/CUQIpy/_images/deconv2D_regularization_parameter.png" alt="Regularization parameter (posterior trace)" width="660">
 </p>
 
-## Plugins
+## 🔌 Plugins
```
[the remark this answers is not in the document]
Comment (nabriis) [reply README.md:77]: Done

File: README.md [hunk 3]
```diff
@@ -6,70 +6,82 @@
...
 CUQIpy can be extended with additional functionality by installing optional plugins. These can be found at
 [CUQI-DTU](https://github.com/CUQI-DTU?q=CUQIpy-).
 
-## Contributing
+## 🤝 Contributing
 We welcome contributions to CUQIpy. Please see our [contributing guidelines](https://cuqi-dtu.github.io/CUQIpy/dev/index.html) for more information.
 
-## Contributors
+## 🌟 Contributors
```
[the remark this answers is not in the document]
Comment (nabriis) [reply README.md:84]: Created issue #286

**Review by amal-ghamdi — APPROVED** [review]

**Pull request merged**